players/001-selfsame

```diff
@@ -0,0 +1 @@
+12
```

Comment:
ah i see it https://github.com/audiodude/tiny-nomic/blob/21429fde4eb8455e72743e2946d37aa889677082/mutable_rules/201_Turn-alternation.md